Boost 1.17 seems to require C++17, upstream recommends either using
boost 1.16 or bumping the utilized C++ standard. Let's try the latter
for now.

See https://github.com/badaix/snapcast/issues/902#issuecomment-902601706

diff -upr snapcast-0.25.0.orig/CMakeLists.txt snapcast-0.25.0/CMakeLists.txt
--- snapcast-0.25.0.orig/CMakeLists.txt	2021-10-26 19:01:23.800898558 +0200
+++ snapcast-0.25.0/CMakeLists.txt	2021-10-26 19:02:49.088728542 +0200
@@ -127,7 +127,7 @@ if(NOT ANDROID)
 endif()
 
 # Configure compiler options
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
 
 # Get arch
 include(${CMAKE_SOURCE_DIR}/cmake/TargetArch.cmake)
